aws_topology/stackstate_checks/aws_topology/resources/sqs.py

```diff
@@ -4,16 +4,12 @@
 from schematics.types import StringType, ModelType
 
 
-def get_queue_name_from_url(url):
-    return url.rsplit('/', 1)[-1]
-
-
 def create_arn(region=None, account_id=None, resource_id=None, **kwargs):
     return arn(
         resource='sqs',
-        region=region,
-        account_id=account_id,
-        resource_id=get_queue_name_from_url(resource_id),
+        region=resource_id.split('.', 2)[1],
```

Comment:
Why are we ignoring the `region`, `account_id` that is being passed in? Do the splitting before calling this function if they are indeed different

Reply:
Jacco and I had a discussion and we need to rework how we pass in these values, as using them breaks when the resource exists within another account and region. For now this resolves the issue